### PR TITLE
data loaded to postgres

### PR DESCRIPTION
**Pull request by jchen0506**
Hi, @jensenchelese @Vasanarul @orlando-apps 
Could you review my code of seeding Postgres when you get a chance?
It's currently seeding one css file now and I'll add another query seeding the other csv since it's getting way too large.
Thank you

### CODE REVIEW

File: db/pgseed.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require('dotenv').config();
+
+const { Pool, Client } = require('pg');
+// pools will use environment variables
+// for connection information
+const client = new Client({
+  user: 'jingchen',
```
Comment (orlando-apps) [inline db/pgseed.js:7]: i think would be a good idea to put this information into a file, then gitignore.

File: db/pgseed.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      itemid int,
+      name text,
+      location text,
+      title text,
+      comment text,
+      likes int,
+      dislike int,
+      star int);
+  COPY reviews(itemid,name,location,title,comment,likes,dislike,star)
+  FROM '/Users/jingchen/Documents/HackReactor/Project/SDC/review-service/db/reviews.csv'
```
Comment (orlando-apps) [inline db/pgseed.js:32]: later on, can add fs path and resolve path

**Comment by orlando-apps** [comment]
Overall, looks good.

**Comment by Vasanarul** [comment]
Hi Jing, looks good to me!

**Comment by jchen0506** [comment]
> Overall, looks good.

Thanks for catch that! Good point. I've already modified all of them. Can you review one more time? Thank you

**Pull request merged**